Specs/Scene/ModelSpec.js

```diff
@@ -120,6 +120,7 @@ defineSuite([
 
     var boomBoxUrl = './Data/Models/PBR/BoomBox/BoomBox.gltf';
     var boxPbrUrl = './Data/Models/PBR/Box/Box.gltf';
+    var boxPbrUnlitUrl = './Data/Models/PBR/Box/Box-Unlit.gltf';
     var boxAnimatedPbrUrl = './Data/Models/PBR/BoxAnimated/BoxAnimated.gltf';
```

Comment:
Put the model in it's own folder like we do here. Also use camel case like we do for the other models. The path should be `./Data/Models/PBR/BoxUnlit/BoxUnlit.gltf`. Just for consistency's sake.